src/pages/Integrate.js

```diff
@@ -42,14 +68,16 @@ const IntegrationCard = ({ text, active, color = null }) => (
   >
     <Flex align="center" justifyContent="space-between">
       <Box as={text} size="64px" color={color} />
-      <Button
-        mt={4}
-        type="submit"
-        variantColor={active ? "green" : "teal"}
-        variant={active ? "solid" : "outline"}
-      >
-        {active ? "Connected" : "Connect"}
-      </Button>
+      <a href={url}>
```

Comment:
I think we should use `Link` tag from `chakra-ui` like everywhere else in the code instead of `a` to maintain consistency.

Reply:
```Link``` doesn't apply here because we need an external URL, I tried to use ```Link``` at first.